ledger/evalindexer_test.go

```diff
@@ -305,6 +307,18 @@ func newTestLedger(t testing.TB, balances bookkeeping.GenesisBalances) *Ledger {
 	return l
 }
 
+func newTestLedger(t testing.TB, balances bookkeeping.GenesisBalances) *Ledger {
+	log := logging.TestingLog(t)
+	log.SetLevel(logging.Info)
+	return newTestLedgerWithLogger(t, balances, log)
+}
+
+func newTestFilteredLedger(t testing.TB, balances bookkeeping.GenesisBalances) *Ledger {
+	log := logging.TestingLogWithFilter(t, logging.DBLockedFilter)
+	log.SetLevel(logging.Warn)
+	return newTestLedgerWithLogger(t, balances, log)
+}
```

Comment:
what is the chance that developers might use this and not realize that they legitimately have caused a ledger overlocking issue? Could we maybe make it so that the tests add this filter *only if* flagged in a certain way?

Alternatively, if this message is only ever noise and we don't care about it... why keep it at all?

Reply:
if it fails to lock way to many times it will error.
tests use in-memory sqlite that known to have locking issues but it way faster than on-disk one. the intent is to ignore such messages for in-memory tests.